src/common/bart_builder.cc

```diff
@@ -0,0 +1,76 @@
+#include "bart_builder.h"
+
+#include <unordered_map>
+
+#include <deal.II/fe/fe_dgq.h>
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_raviart_thomas.h>
+
+template <int dim>
+BARTBuilder<dim>::BARTBuilder (dealii::ParameterHandler &prm) {
+  SetParams (prm);
+}
+
+template <int dim>
+BARTBuilder<dim>::~BARTBuilder () {}
```

Comment:
Should this be marked `inline`?

Reply:
What do you mean by "this"? Destructor?

Reply:
Yes. Since we are defining our destructors like this, explicitly and outside of the class, we should consider adding `inline`. If we defined the empty destructor in the class body it would be implicitly `inline`; if we didn't define a destructor (which we don't have to for this class) the generated one would also be `inline`. So should we make this `inline` to provide the same option to the compiler?